vector/src/main/java/im/vector/app/features/settings/locale/LocalePickerController.kt

```diff
@@ -84,6 +90,11 @@ class LocalePickerController @Inject constructor(
                                 }
                             }
                 }
+            is Fail    ->
```

Comment:
was this a missing case? (and the update caught it? :tada: )

Reply:
I added the code for the `when` to be exhaustive, but I think at this place it cannot fail, since we are loading locale data. But who knows?